package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
no semver until 1.0.0 . Simply because I know that there will be lot of breaking changes. Lot of most popular API are already stable. But also lot of less popular but still needed features aren't. I'll go to 1.0.0 once I cover most of features with tests and implement most of features needed to implement plus make refactorings to make a code robust and maintainable I'll go with 1.0.0. You are right that there will be always be stuff to improve or change or add, however right now typeorm is in active development which means we still expect breaking changes. And I don't want 1.0.0 to be come 17.0.0 in one year of development.